src/pkg/repo_install.rs

```diff
@@ -26,25 +28,22 @@ pub fn run(
 
     let (provider, repo_path) = parse_repo_spec(repo_spec)?;
 
-    let repo_file_names = ["zoi.yaml"];
```

Comment:
Not sure if this is intentional, but this was removed since it's only a single entry in the slice and move the filename to a const. Can revert, or change the const to a slice if you plan on adding more later on.